examples/beegfs/config.json

```diff
@@ -0,0 +1,193 @@
+{
+    "location": "variables.location",
+    "resource_group": "variables.resource_group",
+    "install_from": "headnode",
+    "admin_user": "hpcadmin",
+    "variables" : {
+        "image" : "OpenLogic:CentOS:7.6:latest",
+        "hpc_image": "OpenLogic:CentOS-HPC:7.6:latest",
+        "location": "<NOT-SET>",
+        "resource_group": "<NOT-SET>",
+        "vnet_resource_group": "variables.resource_group"
+    },
+    "vnet": {
+        "resource_group": "variables.vnet_resource_group",
+        "name": "hpcvnet",
+        "address_prefix": "10.2.0.0/20",
+        "subnets": {
+            "admin": "10.2.1.0/24",
+            "viz": "10.2.2.0/24",
+            "compute": "10.2.4.0/22"
+        }
+    },
+    "resources": {
+        "headnode": {
+            "type": "vm",
+            "vm_type": "Standard_D8s_v3",
+            "accelerated_networking": true,
+            "public_ip": true,
+            "image": "variables.image",
+            "subnet": "compute",
+            "data_disks": [2048, 2048],
+            "tags": [
+                "cndefault",
+                "nfsserver",
+                "pbsserver",
+                "loginnode",
+                "localuser",
+                "disable-selinux",
+                "beegfspkgs",
+                "beegfsc"
+            ]
+        },
+        "compute": {
+            "type": "vmss",
+            "vm_type": "Standard_HB60rs",
+            "instances": 2,
+            "image": "variables.hpc_image",
+            "subnet": "compute",
+            "tags": [
+                "nfsclient",
+                "pbsclient",
+                "cndefault",
+                "localuser",
+                "disable-selinux",
+                "beegfspkgs",
+                "beegfsc"
+            ]
+        },
+        "beegfsm": {
+            "type": "vm",
+            "vm_type": "Standard_L8s_v2",
+            "accelerated_networking": true,
+            "image": "variables.image",
+            "subnet": "compute",
+            "tags": [
+                "beegfspkgs",
+                "beegfsm",
+                "disable-selinux",
+                "beegfsc"
+            ]
+        },
+        "beegfssm": {
+            "type": "vmss",
+            "vm_type": "Standard_L8s_v2",
+            "instances": 2,
+            "accelerated_networking": true,
+            "image": "variables.image",
+            "subnet": "compute",
+            "tags": [
+                "beegfspkgs",
+                "beegfssd",
+                "beegfsmd",
+                "disable-selinux"
+            ]
+        }
+    },
+    "install": [
+        {
+            "script": "disable-selinux.sh",
+            "tag": "disable-selinux",
+            "sudo": true
+        },
+        {
+            "script": "beegfspkgs.sh",
+            "tag": "beegfspkgs",
+            "reboot": true,
```

Comment:
Do you need a reboot?  You are not updating a kernel as far as I can see

Reply:
No, do not need reboot, I removed it.